napari/layers/points/points.py

```diff
@@ -1272,6 +1277,18 @@ def _set_editable(self, editable=None):
         if not self.editable:
             self.mode = Mode.PAN_ZOOM
 
+        previous_allow_point_add = self._allow_point_add
+        if self.ndim < 3 and self._ndisplay == 3:
+            self._allow_point_add = False
+            self.editable = False
+        elif self._ndisplay == 3:
+            self._allow_point_add = False
+        else:
+            self._allow_point_add = True
+
+        if previous_allow_point_add != self._allow_point_add:
+            self.events.editable()
```

Comment:
This is an implementation that allows us to disable interactive point adding in 3D. If we just want to disable it when `self.ndim < 3 and self._ndisplay == 3`, it could be:

```suggestion
        if self.ndim < 3 and self._ndisplay == 3:
            self.editable = False
```

Reply:
Based on feedback, I have switched to just disallowing interactivity when ndim < 3 and ndisplay == 3 (i.e., point adding is allows for ndim >= 3 layers in 3D mode).